public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -119,6 +132,21 @@ export const LokiQueryBuilderOptions = React.memo<Props>(
               aria-label="Select resolution"
             />
           </EditorField>
+          {config.featureToggles.lokiQuerySplitting && (
+            <EditorField
+              label="Chunk Range"
+              tooltip="Experimental: Defines the range of a single query chunk when query chunking is used."
```

Comment:
I don't really think it's necessary to put "experimental" all over the place. If the user is enabling the feature flag, they should be aware of it. Moreover, this makes it look like the range is experimental, while the whole feature is, or hypothetically is.

```suggestion
              tooltip="Defines the range of a single query chunk when query chunking is used."
```